translations/w3c_website_templates_bundle+intl-icu.zh-hans.yaml

```diff
@@ -1,205 +1,256 @@
+            ja: 日语
+            zh-hans: 简体中文
+    default_title: 万维网联盟 (W3C)
     default_description: >-
-        __page.default_description
-    skip_to_content: __page.skip_to_content
+        万维网联盟 (W3C) 是一个国际标准组织。众多会员组织、全职员工及广大社区通力合作，制定并完善网络标准。
+    skip_to_content: 转到内容
     translations: >-
-        __page.translations
-    translationsarialabel: __page.translations.arialabel
-    toc: __page.toc
-    related_content: __page.related
+        切换语言：
+    translationsarialabel: 翻译
+    toc: 页面内容跳转至链接
+    related_content: 在这个部分 (TODO)
 footer:
     copyright: >-
-        __footer.copyright {date, date, ::y}
+        Copyright © {date, date, ::y} W3C <sup>®</sup> (<a href="https://www.csail.mit.edu/" hreflang="en"><abbr
+        title="麻省理工学院">MIT</abbr></a>, <a
+        href="https://www.ercim.eu/" hreflang="en"><abbr
+        title="欧洲信息学和数学研究联合会">ERCIM</abbr></a>,
+        <a href="https://www.keio.ac.jp/">庆应义塾大学 Keio</a>, <a href="https://ev.buaa.edu.cn/">北京航空航天大学 Beihang</a>) <a
+        href="/Consortium/Legal/ipr-notice">使用章程适用 (TODO)</a>.
     links:
         hreflang: zh-hans
-        home: __footer.links.home
+        home: 主页
         homeurl: /zh-hans/
-        contact: __footer.links.contact
+        contact: 联系人
         contacturl: /zh-hans/contact/
-        help: __footer.links.help
+        help: 帮助
         helpurl: /zh-hans/help/
-        sponsor: __footer.links.sponsor
+        sponsor: 赞助
         sponsorurl: /zh-hans/sponsor/
-        donate: _footer.links.donate
+        donate: 捐赠
         donateurl: /zh-hans/donate/
-        privacy_policy: __footer.links.privacy_policy
+        privacy_policy: 隐私条款
         privacy_policyurl: /zh-hans/policies/privacy/
-        legal: __footer.links.legal
+        legal:  政策法规
         legalurl: /zh-hans/policies/
-        cookies: __footer.links.cookies
+        cookies: Cookie
         cookiesurl: /zh-hans/cookies/
-        system_status: __footer.links.system_status
+        system_status: 系统状态
         system_statusurl: https://status.w3.org/
-        twitter: __footer.links.twitter
-        github: __footer.links.github
+        twitter: W3C 在推特 (TODO)
+        github: W3C 在 GitHub (TODO)
 banner:
     beta:
         text: >-
-            __banner.beta.text
-        icon: __banner.beta.icon
+            这是一个全新的网站，<a href="mailto:team-website-redesign@w3.org">请给予反馈</a>帮助我们建设完善网站。
+        icon: 测试版图标 (TODO)
 header:
-    logo_alt: __header.logo_alt
-    logo_link: __header.logo_link
-    search_link: __header.search_link
-    account_link: __header.account_link
+    logo_alt: W3C 标志
+    logo_link: 访问 W3C 主页
+    search_link: 搜索
+    account_link: 我的账号
 breadcrumbs:
-    home: __breadcrumbs.home
+    home: 主页
 pagination:
-    first: __pagination.first
-    previous: __pagination.previous
-    next: __pagination.next
-    last: __pagination.last
-    page: "__pagination.page {page}"
+    first: 首页
+    previous: 上一<span class="visuallyhidden">页</span>  (TODO visuallyhidden?)
```

Comment:
Hi @jean-gui, could you explain how is `visuallyhidden` used? In what situation can we see it, or not see it?

![image](https://user-images.githubusercontent.com/8498677/162403866-54a46ddc-01a3-4db6-b51a-5f879c821109.png)

Reply:
This class is explained at https://design-system.w3.org/styles/how-to-hide-and-show-things.html#visually-hiding-items. It visually hides content while keeping it accessible to screen readers. So in this case, a screen reader would say "Next page".